apps/src/templates/instructions/HelpTab.jsx

```diff
@@ -0,0 +1,59 @@
+import {showVideoDialog} from "../../code-studio/videos";
+import React, {PropTypes} from 'react';
+
+var styles = {
+  referenceArea: {
+    marginTop: 20,
+  },
+  videoLink: {
+    display: 'inline-block',
+    margin: 8,
+    fontWeight: 'bold',
+    fontSize: 16,
+    lineHeight: 25 + 'px',
+  },
+  videoThumbnail: {
+    borderRadius: 5,
+    height: 40,
+    width: 'auto',
+    marginRight: 8
+  }
+};
+
+const HelpTab = React.createClass({
+  propTypes: {
+    videoData: PropTypes.object
```

Comment:
should be required? we should also use `PropTypes.shape` to describe the fields we expect

Reply:
I'm going to leave un-required since there might be levels for which we will have other resources (coming soon in this tab) but no videos.

Reply:
Right now if there's no videoData, we're going to throw an exception when we do things like `videoData.name`. If videoData really is optional, perhaps the right thing is to replace our anchor tag with 
`{videoData && <a ... />}`

Reply:
Got it - I see where the issue is.